camus-shopify/src/main/scala/com/linkedin/camus/shopify/LateArrivingDataMonitor.scala

```diff
@@ -76,8 +76,11 @@ object LateArrivingDataMonitor {
           violationFound = true
           log.error(s"Violation found in ${camusDrop.path}")
           try {
-            val count = camusDrop.violationCount
-            log.error(s"=> late-arriving records for dir ${camusDrop.topicDir} in ${camusDrop.path}: $count")
+            val violationList = camusDrop.violations
+            violationList.foreach({
+              fileAndCount =>
+                log.error(s"=> late-arriving records in file ${fileAndCount._1}: ${fileAndCount._2}")
+            })
             StatsdReporter.gauge(properties, "late-arriving-data", 1L, s"directory:${camusDrop.topicDir}")
```

Comment:
waat, I'd forgotten about this, we have it in datadog too?

Reply:
can we add the count to datadog?

Reply:
We can. We'd just have to add extra logic to aggregate by directory. Is there value in doing this? We would still need to check Splunk for hourly dir breakdown.

Reply:
True. Splunk is good enough.